makefile

```diff
@@ -387,6 +385,15 @@ scramble:
 	python cmd/scrambled-eggs/scramble.py $(file) $(multiarg)
 
 # mongodb utility targets
+mongotoolsDir := mongotools
+run-mongotools = $(if $(wildcard $(mongotoolsDir)), ./$(mongotoolsDir)/$(1), $(1))
```

Comment:
Same thing as above - would help to explain in words what this is doing (using the locally-downloaded mongo-tools if possible, and otherwise using the globally-installed relevant mongo-tool executable?

Reply:
Done.